libs/btp/navigation/navigation-list/navigation-list-item.component.ts

```diff
@@ -253,6 +259,21 @@ export class NavigationListItemComponent
                 .join(' ');
             this._popoverService.updateContent(null, this.childrenTemplate());
         });
+
+        effect(
+            () => {
+                const activeRoute = this.routerLinkActive();
+
+                activeRoute?.isActiveChange
+                    .pipe(startWith(activeRoute.isActive), takeUntilDestroyed(this._destroyRef))
+                    .subscribe((isActive) => {
+                        this.isRouterLinkActive.set(isActive);
+                    });
+            },
+            {
+                allowSignalWrites: true
+            }
+        );
```

Comment:
you could do `toSignal` with `Injector` passed to it and not require `allowSignalWrites`

Reply:
Yeah, tried that, didn't work actually